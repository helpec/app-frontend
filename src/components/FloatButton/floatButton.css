.panico{
    position: fixed;
    bottom: 0;
    right: 0;
    margin: 50px;
    color: white;
    background: red;
    border: none;
    padding: 10px 20px;
    border-radius: 20px;
    font-weight: bold;
}