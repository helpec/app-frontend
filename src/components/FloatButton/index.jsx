import React, { Component } from 'react';
import './floatButton.css'

class FloatButton extends Component {
  onclick= () => { window.open ("https://wa.me/5511982326391?text=Estou%20passando%20mal%2C%20voc%C3%AA%20pode%20me%20ajudar%3F")
    }
  
  render() {
    return (
        <button className= "panico" type= "button" onClick={this.onclick}> PÃ¢nico</button> 
        
    )
  }
}

export default FloatButton;