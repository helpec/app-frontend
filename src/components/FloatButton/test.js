import React from 'react';
import renderer from 'react-test-renderer';

import FloatButton from '.';

test('Testar componente de FloatButton da aplicação', () => {

    const component = renderer.create(
      <FloatButton />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

});
