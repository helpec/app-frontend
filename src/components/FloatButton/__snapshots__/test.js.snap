// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testar componente de FloatButton da aplicação 1`] = `
<button
  className="panico"
  onClick={[Function]}
  type="button"
>
   Pânico
</button>
`;
